src/scripts/main.js

```diff
@@ -2,7 +2,27 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
-
 // write your code here
+const table = document.getElementsByClassName('dashboard')[0];
+const getCentury = (deathAge) => {
+  return Math.ceil(deathAge / 100);
+};
+
+const getFullGender = (sex) => {
+  return sex === 'f' ? 'Female' : 'Male';
+};
```

Comment:
```suggestion
const getFullGender = sex => sex === 'f' ? 'Female' : 'Male';
```

You can use implicit return for the arrow function.